starknet_py/utils/contructor_args_translator.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+
+from starkware.starknet.public.abi_structs import identifier_manager_from_abi
+
+from starknet_py.utils.data_transformer import FunctionCallSerializer
+
+
+def translate_constructor_args(abi: list, constructor_args: any) -> List[int]:
+    constructor_abi = next(
+        (member for member in abi if member["type"] == "constructor"),
+        None,
+    )
+
+    # Constructor might not accept any arguments
+    if not constructor_abi or not constructor_abi["inputs"]:
```

Comment:
So we check two cases here:
1. Contract doesn't have a constructor
2. There is a constructor with no arguments

Right?

Reply:
Not sure, because it was moved from the Contract to avoid conflicts